program/src/instruction.rs

```diff
@@ -96,9 +107,12 @@ pub enum Instructions {
     ///
     ///   0. `[writable]` New or existing account storing verified messages
     ///   1. `[]` Reward manager
+    ///   1. `[]` Reward manager authority
+    ///   1. `[signer]` Funder
     ///   2. `[]` Sender
-    ///   3. `[]` Sysvar instruction id
-    VerifyTransferSignature,
+    ///   8. `[]` Sysvar rent
+    ///   2. `[]` Instruction info
```

Comment:
looks like this is missing system_program here, if the accounts in the verify_transfer_signature method are correct